lighty-core/lighty-controller/src/test/java/io/lighty/core/controller/util/FileToDatastoreUtilsTest.java

```diff
@@ -37,19 +40,29 @@
 public class FileToDatastoreUtilsTest {
 
     private static final String INITIAL_CONTAINER_PATH = "/data/container-value-1.json";
+    private static final String CASE_CONTAINER_PATH = "/data/case-container-value.json";
     private static final String OVERRIDE_CONTAINER_PATH = "/data/container-value-2.xml";
     private static final String OVERRIDE_VALUE_JSON_PATH = "/data/leaf-value-3.json";
     private static final String OVERRIDE_VALUE_XML_PATH = "/data/leaf-value-4.xml";
     private static final String MULTIPLE_TOP_JSON_PATH = "/data/multiple-top-element.json";
     private static final String MULTIPLE_TOP_XML_PATH = "/data/multiple-top-element.xml";
     private static final InstanceIdentifier<TopLevelContainer> TOP_LEVEL_CONTAINER_IID
             = InstanceIdentifier.create(TopLevelContainer.class);
+
     private static final InstanceIdentifier<SampleList> SAMPLE_LIST_ID1_IID
             = InstanceIdentifier.builder(SampleList.class, new SampleListKey("ID1")).build();
     private static final InstanceIdentifier<SampleList> SAMPLE_LIST_ID2_IID
             = InstanceIdentifier.builder(SampleList.class, new SampleListKey("ID2")).build();
 
     private static final YangInstanceIdentifier ROOT_YII = YangInstanceIdentifier.empty();
+
+    private static final InstanceIdentifier<ChoiceContainer> CHOICE_CONTAINER_YII
+            = InstanceIdentifier.builder(ChoiceContainer.class).build();
+
+    private static final YangInstanceIdentifier INNER_CASE_YIID = YangInstanceIdentifier.create(
+            YangInstanceIdentifier.NodeIdentifier.create(ChoiceContainer.QNAME),
```

Comment:
Add static import for `YangInstanceIdentifier.NodeIdentifier`.

Reply:
added